orx-g-code/src/jvmTest/kotlin/TestGenerator.kt

```diff
@@ -0,0 +1,65 @@
+package test
+
+import org.amshove.kluent.`should be equal to`
+import org.openrndr.extra.gcode.Command
+import org.openrndr.extra.gcode.roundedTo
+import org.openrndr.extra.gcode.withoutDuplicates
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import kotlin.test.assertContentEquals
+
+object TestGenerator : Spek({
```

Comment:
We're actually (slowly) in the process of [removing usages of Spek and using kotlin.test instead](https://github.com/openrndr/openrndr/blob/master/CONTRIBUTING.md#writing-tests). Would you be able to convert these tests so they follow the style of the [tests written in openrndr-svg](https://github.com/openrndr/openrndr/blob/master/openrndr-svg/src/jvmTest/kotlin/org/openrndr/svg/TestSVGLoader.kt) for example?